### ISSUE TITLE
[mono] mono 4.2.2 breaks with 'segmentation fault'

### ISSUE THREAD

**Issue by mediocaballero** [SynoCommunity/spksrc#2298]
Issue description:
I need a more up-to-date version of mono than the official Synology beta (4.2.1).
Tried SynoCommunity one (reported as 4.2.2). It downloads, installs and seems to be running, but whenever I try to run something (even just checking the version) it breaks with a 'segmentation fault' error.

Console output:

```
admin@NAS:/usr/local/mono/bin$ ./mono --version
Segmentation fault
```

Model: DS215j
Architecture: MARVELL Armada 375 88F6720
DSM version: DSM 6.0-7321 Update 3
Log file: n/a

**Comment by Dr-Bean** [comment]
Please follow the guidelines when opening a new issue: https://github.com/SynoCommunity/spksrc/blob/master/CONTRIBUTING.md#issue-content

**Comment by mediocaballero** [comment]
Updated following guidelines. Thanks!

**Comment by Dr-Bean** [comment]
Waiting on #2296.

**Comment by Dr-Bean** [comment]
I've published a new set of Mono 4.6.2.7-8 packages for DSM6, available on the beta channel (maybe not immediately, but it should show up at some point)
Maybe one of you can check if the segfault issue is solved with those packages.
Note that the package won't show up as an update in Package Center, the previous package will have to be removed first.

**Comment by Safihre** [comment]
Mono is now at 5.8.0, so assuming these issues to be fixed.
